src/main/java/org/mastodon/mamut/nearest/ui/NearestObjectStatMainPanel.java

```diff
@@ -0,0 +1,132 @@
+package org.mastodon.mamut.nearest.ui;
+
+import java.awt.BorderLayout;
+
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.JSplitPane;
+import javax.swing.border.EmptyBorder;
+
+import org.mastodon.app.MastodonIcons;
+import org.mastodon.feature.ui.AvailableFeatureProjections;
+import org.mastodon.mamut.nearest.NearestObjectStatModel;
+import org.mastodon.mamut.nearest.NearestObjectStatModel.NearestObjectStatItem;
+import org.mastodon.ui.ProgressListener;
+
+public class NearestObjectStatMainPanel extends JPanel
+{
+
+	private static final long serialVersionUID = 1L;
+
+	final JButton btnAdd;
+
+	final JButton btnCompute;
+
+	final JButton btnCancel;
+
+	final JLabel lblLog;
```

Comment:
The variables are difficult to read for others who are not used to "lbl" = "label". Could you use "labelLog" etc. instead?